actionview/lib/action_view/helpers/asset_tag_helper.rb

```diff
@@ -269,6 +270,8 @@ def image_alt(src)
       #   # => <video preload="none" controls="controls" src="/videos/trailer.ogg" ></video>
       #   video_tag("trailer.m4v", size: "16x10", poster: "screenshot.png")
       #   # => <video src="/videos/trailer.m4v" width="16" height="10" poster="/assets/screenshot.png"></video>
+      #   video_tag("trailer.m4v", size: "16x10", poster: "screenshot.png", public_poster_folder: true)
```

Comment:
There need to be 2 options here as the video may be in the pipeline but the poster might not be. Good catch though on the naming. I'll fix this.